### PR TITLE
[ MB-12420 ] Add the UpdateMTOReviewedBillableWeightsAt event

### PR DESCRIPTION
**Pull request by rogeruiz**
## Summary

As an office user, I want to have the ability to see when a TIO clicks on the review weights button max  so that the office users can easily see if the TIO has reviewed the weights. 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Run the client tests locally

```sh
make client_test
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Make sure that client tests are passing. Specifically tests found in `src/constants/moveHistoryEventTemplate.test.js`
2. Login as a TIO and click on the **Review weights** button.
3. Check the history log for an event that matches **Updated move: Reviewed weights**

### CODE REVIEW

File: src/constants/moveHistoryEventTemplate.js [hunk 1]
```diff
@@ -492,6 +492,15 @@ export const undefinedEvent = buildMoveHistoryEventTemplate({
   },
 });
 
+export const updateMTOReviewedBillableWeightsAt = buildMoveHistoryEventTemplate({
+  action: dbActions.UPDATE,
+  eventName: moveHistoryOperations.updateMTOReviewedBillableWeightsAt,
+  tableName: 'moves',
```
Comment (michaelvaldes-truss) [inline src/constants/moveHistoryEventTemplate.js:498]: **suggestion**: we can use the dbTables constant here instead of a string

**Review by michaelvaldes-truss — CHANGES_REQUESTED** [review]
Testing worked for me, just left a suggestion!
image

File: src/constants/moveHistoryEventTemplate.js (hunk 1 above)
Comment (rogeruiz) [reply src/constants/moveHistoryEventTemplate.js:498]:
```suggestion
  tableName: dbTables.moves,
```

Comment (rogeruiz) [reply src/constants/moveHistoryEventTemplate.js]: @michaelvaldes-truss I encourage you to use the GitHub suggestion feature next time for changes. I wasn't aware we switched to constants. Thank you for this review.

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm!

**Pull request merged**